src/Microsoft.CodeAnalysis.Razor.Workspaces/DefaultRazorSyntaxFactsService.cs

```diff
@@ -139,21 +135,21 @@ public override IReadOnlyList<TagHelperSpan> GetTagHelperSpans(RazorSyntaxTree s
             return results;
         }
 
-        public override int? GetDesiredIndentation(RazorSyntaxTree syntaxTree, ITextSnapshot syntaxTreeSnapshot, ITextSnapshotLine line, int indentSize, int tabSize)
+        public override int? GetDesiredIndentation(RazorSyntaxTree syntaxTree, int previousLineEndIndex, Func<int, string> getLineContent, int indentSize, int tabSize)
```

Comment:
This is probably an example of a case where we would just split this particular API out into its own service.